spec/features/course/achievement_management_spec.rb

```diff
@@ -6,6 +6,14 @@
   with_tenant(:instance) do
     let(:user) { create(:administrator) }
     let(:course) { create(:course) }
+    let(:other_achievement) { create(:course_achievement, course: course) }
+    let(:achievement_condition) { create(:course_condition_achievement,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.